modules/library/referencing/src/main/java/org/geotools/referencing/crs/DefaultGeographicCRS.java

```diff
@@ -257,4 +260,23 @@ protected String formatWKT(final Formatter formatter) {
         formatter.setAngularUnit(oldUnit);
         return "GEOGCS";
     }
+
+    @Override
+    public String formatPROJ(PROJFormatter formatter) {
+        if (!formatter.isProjectedCRS()) {
+            formatter.append("+proj=longlat ");
+        }
+        final Unit<Angle> oldUnit = formatter.getAngularUnit();
+        final Unit<Angle> unit = getAngularUnit(coordinateSystem);
+
+        formatter.setAngularUnit(unit);
+        if (datum instanceof PROJFormattable) {
+            formatter.append((PROJFormattable) datum);
+        }
+
+        formatter.append((PROJFormattable) ((GeodeticDatum) datum).getPrimeMeridian());
+        formatter.setAngularUnit(oldUnit);
+
+        return "";
```

Comment:
This is... surprising. What is the meaning of the empty string? Assigning it to a constant would help document and thus understand it.